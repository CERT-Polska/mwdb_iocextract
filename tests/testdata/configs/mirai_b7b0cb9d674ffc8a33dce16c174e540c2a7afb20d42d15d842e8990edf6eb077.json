{"cncs": [{"host": "23.254.215.229", "port": 1337}], "table_key": "0xdedefbaf", "variant": "SORA", "type": "mirai"}