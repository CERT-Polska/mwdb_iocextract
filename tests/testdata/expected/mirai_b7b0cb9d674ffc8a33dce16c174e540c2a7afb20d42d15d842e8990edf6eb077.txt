NetLoc 23.254.215.229:1337
